### ISSUE TITLE
Parse Transaction:  Parsing Unsigned Transactions  Fails: TestNet 

### ISSUE THREAD

**Issue by AussieDelly** [PoC-Consortium/burstcoin#395]
According to the documentation the Parse Transaction API should be able to Parse an Unsigned Transaction.

It appears to fail with:- {"errorDescription":"Incorrect request","errorCode":1}

API DOCUMENTATION

**Parse Transaction**
Get a transaction object given a (signed or **_unsigned_**) transaction bytecode, or re-parse a transaction object. Verify the signature.

**Request:**
requestType is parseTransaction
transactionBytes is the signed or unsigned bytecode of the transaction (optional)
transactionJSON is the transaction object (optional if transactionBytes is included)
Response: Refer to Get Transaction for additional fields.

(B) verify is true if the signature is verified, false otherwise

An example from the Test Net:-

https://wallet.dev.burst-test.net/burst?requestType=parseTransaction&transactionBytes=0314e0b0b4073c00ec8764b4593a2e1db2db71177c8f5639f55ceac7d6616fab30d635bc2fb9bc61a014f4d3ee4ff7df000000000000000000e1f5050000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000ba960100b23ba8cbecce3dc101fb220c1851636bcae8030000a025260000000000f8d1be0801b30000807b226d65737361676554797065223a2022506f6c696379222c2276657273696f6e223a20312c226c61746974756465223a202d33312e393532373132312c226c6f6e676974756465223a203131352e383630343739362c22636f766572537461727473223a202253756e2c2031362053657020323031382030303a30303a303020474d54222c22636f766572456e6473223a20224d6f6e2c203138204d617220323031392030303a30303a303020474d54227d

**Comment by umbrellacorp03** [comment]
We will take a closer look at BRS code, but indeed it seems impossible to parse usigned transaction bytes for now...

For the moment, I corrected the API docs on burstwiki.org to reflect this situation.

I will get back to you if I have any news (if it really was a bug or deprecated docs).

In the meantime, it is possible to get the signed transaction bytes using [SignTransaction](https://burstwiki.org/wiki/The_Burst_API#Sign_Transaction).

Sorry for the inconvenience caused and thank you very much for noticing this.

**Comment by AussieDelly** [comment]
Yeah thats what I'm doing at the moment. I wanted to send a bill to a customer via an unsigned transaction in a burst message. Its a bit ugly for the customer if they have to provide their pass phrase to read the bill. https://aussiedelly.wixsite.com/insure/payments. Hopefully it can be fixed.
